lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,15 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
+
+    my $con = $ipc->{bus}->get_connection;
+    my $msg = $con->make_method_call_message("org.opensuse.openqa.Scheduler", "/Scheduler", "org.opensuse.openqa.Scheduler", "emit_jobs_available");
```

Comment:
You need to add `use Net::DBus::Annotation` and it's not `dbus_call_async`, but you want `dbus_call_noreply`. See http://search.cpan.org/~danberr/Net-DBus-1.0.0/lib/Net/DBus/Annotation.pm